### PR TITLE
Netlify Deploy Button for Kodadot-app #1397

### LINKED ISSUE #1397: Create kodadot-app Netlify Deploy Button
Let's put stuff up to bring forking kodadot faster.
Be mindful of all environments if everything works well

- List KodaDot here https://templates.netlify.com/
- When deploy happen, make PR should be listed in #1171 

### Ref
- https://www.netlify.com/blog/2016/11/29/introducing-the-deploy-to-netlify-button/
- https://css-tricks.com/hack-the-deploy-to-netlify-button-using-environment-variables-to-make-a-customizable-site-generator/

### PR DESCRIPTION
**Pull request by AshutoshSingh72**
Thank you for your contribution to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz/).

👇 __ Let's make a quick check before the contribution.
## Problems
I tried to list Kodadot NFT gallery tamplate on  https://www.netlify.com/integrations/templates/
But https://github.com/netlify/templates this Template Git hub Repo is Archived but i tried to make pull request but because it is archived i wan't able to pull 
IMG_20231004_172752

View my Template 
md file https://github.com/AshutoshSingh72/templates/blob/master/src/site/template/Kodadot-nft-gallery.md

## Fixes
But i was able able to build the netlify deploy button added it into the contributor md

## PR Type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #1397
- [x] ## Did your issue had any of the "$" label on it?
- [x] My DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=158VTDxZJidzsMcAdh7h828domveRvpmk1SCxwLUEHhcgY4v&usdamount=0&donation=true)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ We might give retro-active reward, where the bounty label wasn't present, **if w
 
 **For better coordination, please join our [Development channel (#coordination) on [KodaDot Ecosystem Telegram](https://t.me/kodadot_eco)**
 
+## Deploy Kodadot nft gallery to Netlify
+<a href="https://app.netlify.com/start/deploy?repository=https://github.com/kodadot/nft-gallery"><img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify"></a>
+
```
Comment (vikiival) [inline CONTRIBUTING.md:15]:
```suggestion
[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/kodadot/nft-gallery)
```

**Review by vikiival — CHANGES_REQUESTED** [review]
https://www.netlify.com/blog/2021/12/26/deploying-with-the-click-of-a-button/

**Comment by AshutoshSingh72** [comment]
> https://www.netlify.com/blog/2021/12/26/deploying-with-the-click-of-a-button/

😅 Thanks viki i was using the same code but i forgot to update https://github.com/AshutoshSingh72/nft-gallery/commit/7c3d79dc031462fa54bb5284dc6c2dbe254be258

**Comment by AshutoshSingh72** [comment]
> https://www.netlify.com/blog/2021/12/26/deploying-with-the-click-of-a-button/

Updated

**Comment by AshutoshSingh72** [comment]
> https://www.netlify.com/blog/2021/12/26/deploying-with-the-click-of-a-button/

I'm requesting review from you viki

**Comment by AshutoshSingh72** [comment]
Cc @yangwao @vikiival

**Comment by vikiival** [comment]
Hello,
the solution that you provided does not work ^-^
Please take high issue with a higher priority

Screenshot 2023-10-09 at 12 08 49

**Comment by AshutoshSingh72** [comment]
> Hello, the solution that you provided does not work ^-^ Please take high issue with a higher priority
> 
> Screenshot 2023-10-09 at 12 08 49

I'm using the same deploy button and it is working https://6523f73dadb3140a638f697e--melodic-cheesecake-9489e5.netlify.app/

**Comment by AshutoshSingh72** [comment]
> Hello, the solution that you provided does not work ^-^ Please take high issue with a higher priority
> 
> Screenshot 2023-10-09 at 12 08 49

Screenshot_2023-10-09-18-36-31-80

**Comment by vikiival** [comment]
Looks a residual from some old repository 

https://app.netlify.com/sites/joyful-douhua-7de1e0/deploys/6523fb58dae19f089b306ed0

please anyway open new pr with correct commit description.

**Comment by vikiival** [comment]
Yup looks like its working

Screenshot 2023-10-09 at 15 14 03

I had some residual repository.

Reasons Why I closed this PR:
Sir well you do not follow the contributing standards 🙂 

1. 7 same commits
2. Not using PR template
3. Screenshots not provided
4. Issue never created for this PR.

**Comment by AshutoshSingh72** [comment]
> Yup looks like its working
> 
> Screenshot 2023-10-09 at 15 14 03
> I had some residual repository.
> 
> Reasons Why I closed this PR: Sir well you do not follow the contributing standards 🙂
> 
> 1. 7 same commits
> 2. Not using PR template
> 3. Screenshots not provided
> 4. Issue [Create kodadot-app Netlify Deploy Button #1397](https://github.com/kodadot/nft-gallery/issues/1397) was created almost **2yrs ago**

bro for me i'm working here to solve some problems i faced first time joining kodadot contributors so it might be the oldest issue but it is a big deal for new contributors who stuck on this

**Pull request closed without merging**